src/utils.rs

```diff
@@ -242,12 +207,7 @@ pub fn run_pass_manager(context: &Context, module: &mut Module) -> Result<(), Er
     pass_manager.enable_verifier(true);
     pass_manager.add_pass(pass::transform::create_canonicalizer());
     pass_manager.add_pass(pass::conversion::create_scf_to_control_flow());
-    pass_manager.add_pass(pass::conversion::create_arith_to_llvm());
-    pass_manager.add_pass(pass::conversion::create_control_flow_to_llvm());
-    pass_manager.add_pass(pass::conversion::create_index_to_llvm());
-    pass_manager.add_pass(pass::conversion::create_finalize_mem_ref_to_llvm());
-    pass_manager.add_pass(pass::conversion::create_func_to_llvm());
-    pass_manager.add_pass(pass::conversion::create_reconcile_unrealized_casts());
+    pass_manager.add_pass(pass::conversion::create_to_llvm());
```

Comment:
Does this pass also include the canonicalizer and `scf_to_cf`? If so then we can remove those.
Also does it have a canonicalize at the end? I think it does but just in case.

Reply:
it doesn't (scf to cf), which is why its there